components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java

```diff
@@ -221,7 +221,7 @@ public void generateEntitlementPolicies(APIIdentifier apiIdentifier) throws
      * @param policyId Entitlement policy id
      * @return Entitlement policy content
      */
-    String getEntitlementPolicy(String policyId);
+    String getEntitlementPolicy(String policyId) throws AppManagementException;
```

Comment:
Fixed Java doc comment issue in https://github.com/wso2/carbon-appmgt/pull/539/files